src/site/markdown/server/manage-webadmin.md

```diff
@@ -425,3 +425,70 @@ Note that several calls to this endpoint will be run in a sequential pattern.
 
 If the server restarts during the migration, the migration is silently aborted.
 
+## Creating address group
+
+You can use **webadmin** to define address groups.
+
+When a specific email is sent to the group mail address, every group member will receive it.
+
+This feature uses [Recipients rewrite table]() and requires the [RecipientRewriteTable mailet]() to be configured.
+
+### Listing groups
+
+```
+curl -XGET http://ip:port/address/groups
+```
+
+Will return the groups as a list of JSON Strings representing mail addresses. For instance:
+
+```
+["group1@domain.com", "group2@domain.com"]
+```
+
+Response codes:
+ - 200: Success
+ - 500: Internal error
+
+### Listing members of a group
+
+```
+curl -XGET http://ip:port/address/groups/group@domain.com
+```
+
+Will return the group members as a list of JSON Strings representing mail addresses. For instance:
+
+```
+["member1@domain.com", "member2@domain.com"]
+```
+
+Response codes:
+ - 200: Success
+ - 400: Group structure is not valid
+ - 404: The group does not exist
+ - 500: Internal error
+
+### Adding a group member
+
+```
+curl -XPUT http://ip:port/address/groups/group@domain.com/member@domain.com
+```
+
+Will add member@domain.com to group@domain.com
```

Comment:
Should explain that it will create the group if it does not exist

Reply:
ok